packages/server-wallet/src/protocols/state.ts

```diff
@@ -92,9 +109,71 @@ export const toChannelResult = (channelState: ChannelState): ChannelResult => {
 
   const allocations = serializeAllocation(checkThat(outcome, isAllocation));
 
-  return {appData, appDefinition, channelId, participants, turnNum, allocations, status};
+  return {
+    appData,
+    appDefinition,
+    channelId,
+    participants,
+    turnNum,
+    allocations,
+    status,
+    fundingStatus: directFundingStatus,
+  };
 };
 
+export function directFundingStatus(
+  supported: SignedStateVarsWithHash | undefined,
+  fundingFn: (address: Address) => ChannelStateFunding,
+  myParticipant: Participant,
+  fundingStrategy: FundingStrategy
+): FundingStatus {
+  if (fundingStrategy !== 'Direct') {
+    return 'Uncategorized';
+  }
+
+  const outcome = supported?.outcome;
+  if (!supported || !outcome) {
+    return 'Uncategorized';
+  }
+
+  const {allocationItems, assetHolderAddress} = checkThat(outcome, isSimpleAllocation);
+
+  // Collapse all allocation items with my destination into one
```

Comment:
I think the contracts assume that there is only one allocation item per destination, and therefore ignore items after the first (@geoknee could you confirm?). The wallet should therefore do the same.

Reply:
Yes that is the case for `claimAll` and `transfer` (and `claim` #2932). It happens that `transferAll` does not rely on such an assumption. 

It seems like a big assumption to make so I think we should a) be sure that we really need or want to make it and b) if we do, put it very prominently in the docs, advise `ForceMoveApps` to check it*, check it in the wallet when we do ledger funding etc.

*Similar to:
https://github.com/statechannels/statechannels/blob/76ad4a64a9f4bdc02f05afe121ea9cb5d3aaeca7/packages%2Fnitro-protocol%2Fcontracts%2Fexamples%2FSingleAssetPayments.sol#L61-L64